pod/perlop.pod

```diff
@@ -2456,7 +2456,7 @@ X<qx> X<`> X<``> X<backtick>
 =item C<`I<STRING>`>
 
 A string which is (possibly) interpolated and then executed as a
-system command, via F</bin/sh> or its equivalent if required.  Shell
+system command, via F</bin/sh> (not changable) or its equivalent if required.  Shell
```

Comment:
`s/changable/changeable/`

Reply:
I actually looked that up several times first I thought it needed an i, and I almost got it right. Well please go ahead and fix it for me because I'll  only screw it up more thanks!